aiohttp/connector.py

```diff
@@ -954,7 +970,9 @@ def _get_fingerprint(self, req: "ClientRequest") -> Optional["Fingerprint"]:
 
     async def _wrap_create_connection(
         self,
-        *args: Any,
+        factory: functools.partial[ResponseHandler],
```

Comment:
The actual requirement is this: https://github.com/python/typeshed/blob/main/stdlib/asyncio/base_events.pyi#L117

So, I'd suggest doing the same thing with `Callable[[], _ResponseHandlerT]`. Then we can also update the return type so this becomes generic.